addons/common/init_perFrameHandler.sqf

```diff
@@ -15,23 +15,36 @@ GVAR(nextFrameNo) = diag_frameno + 1;
 GVAR(nextFrameBufferA) = [[[], {GVAR(nextFrameNo) = diag_frameno;}]];
 GVAR(nextFrameBufferB) = [];
 GVAR(waitUntilAndExecArray) = [];
-
+GVAR(deletedPFHIndices) = [];
 // per frame handler system
 [QFUNC(onFrame), {
     private _tickTime = diag_tickTime;
     call FUNC(missionTimePFH);
 
     // Execute per frame handlers
     {
-        _x params ["_function", "_delay", "_delta", "", "_args", "_handle"];
+        _x params ["_function", "_delay", "_delta", "", "_args", "_handle", "_isDeleted"];
 
-        if (diag_tickTime > _delta) then {
+        if (diag_tickTime > _delta && !_isDeleted) then {
```

Comment:
this will be called multiple times each frame, lets invert the logic on the var and skip the not
`_isActive`?

Reply:
good point